com.unity.multiplayer.mlapi/CHANGELOG.md

```diff
@@ -1,13 +1,85 @@
+  - `NetworkVariable`s are now tick-based using the `NetworkTickSystem`, tracking time through network interactions and syncs.
+
+- Added message batching to handle consecutive RPC requests sent to the same client. `MessageBatcher` sends batches based on requests from the `RpcQueueProcessing`, by batch size threshold or immediately.
+- [GitHub 494](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/494): Added a constraint to allow one `NetworkObject` per `GameObject`, set through the `DisallowMultipleComponent` attribute.
+- Integrated MLAPI with the Unity Profiler for versions 2020.2 and later.
+- A test project is available for building and experimenting with MLAPI features. This project is located in the package `testproject` folder ([GitHub folder](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/tree/release/0.1.0/testproject)). 
+- Added a [MLAPI Community Contributions](https://github.com/Unity-Technologies/mlapi-community-contributions/tree/master/com.mlapi.contrib.extensions) new GitHub repository to accept extensions from the MLAPI community. Current extensions include moved MLAPI features for lag compensation (useful for Server Authoritative actions) and `TrackedObject`.
+
+### Changes
+
+- [GitHub 520](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/520): MLAPI now uses the Unity Package Manager for installation management.
+- Added functionality and usability to `NetworkVariable`, previously called `NetworkVar`. Updates enhance options and fully replace the need for `SyncedVar`s. 
+- [GitHub 507](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/507): Reimplemented `NetworkAnimator`, which synchronizes animation states for networked objects. 
+- GitHub [444](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/444) and [455](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/455): Channels are now represented as bytes instead of strings.
+
+For users of previous versions of MLAPI, this release renames APIs due to refactoring. All obsolete marked APIs have been removed as per [GitHub 513](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/513) and [GitHub 514](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/514).
+
+| Previous MLAPI Versions | V 0.1.0 Name |
+| -- | -- |
+| `NetworkingManager` | `NetworkManager` |
+| `NetworkedObject` | `NetworkObject` |
+| `NetworkedBehaviour` | `NetworkBehaviour` |
+| `NetworkedClient` | `NetworkClient` |
+| `NetworkedPrefab` | `NetworkPrefab` |
+| `NetworkedVar` | `NetworkVariable` |
+| `NetworkedTransform` | `NetworkTransform` |
+| `NetworkedAnimator` | `NetworkAnimator` |
+| `NetworkedAnimatorEditor` | `NetworkAnimatorEditor` |
+| `NetworkedNavMeshAgent` | `NetworkNavMeshAgent` |
+| `SpawnManager` | `NetworkSpawnManager` |
+| `BitStream/BitSerializer`... | `NetworkBuffer/NetworkSerializer`... |
```

Comment:
```suggestion
| `BitStream` | `NetworkBuffer`|
| `BitWriter` | `NetworkWriter`|
| `BitReader` | `NetworkReader`|
```

Reply:
@MFatihMAR NetworkSerializer/BitSerializer hasn't been part of the last release so not putting it into the changelog. Does that make sense?

Reply:
yeah, this is not a name change. this is a new code/feature we implemented.
I still think it should be in the changelog though, perhaps not under renaming but on the new features added section.

Reply:
@lkrell

Reply:
On it thanks!